src/utils/file.ts

```diff
@@ -105,6 +105,7 @@ export async function getAllFilesFromCurrentPath(
 export async function zipDirectory(
     sourceDir: string,
     outPath: string,
+    includeHiddenFiles: boolean = false,
```

Comment:
I think this should be `true` by default.

So before this change `dot: true`, hence all of the previous method calls assume that hidden files will be included by default, right?

Reply:
Doesn't matter. I changed in all places where this is used. The default value is for future usage and I think false is a more safe default value.

Reply:
It's okay to let it be false, but can you also update the following references:

https://github.com/Genez-io/genezio/blob/dev/src/commands/bundle.ts#L117
https://github.com/Genez-io/genezio/blob/dev/src/bundlers/dart/dartBundler.ts#L214